lib/stdlib/doc/src/sofs.md

```diff
@@ -192,42 +205,44 @@ the case in this module), this is denoted B = \{x : S(x)\}.
   (x\[1], ..., x\[n]) in R and for some (y\[1], ..., y\[m]) in S such that
   x\[i] = y\[j]\}.
 
-- [](){: #sets_definition } The sets recognized by this module are represented
-  by elements of the relation Sets, which is defined as the smallest set such
-  that:
+## Sets handled by this module
 
-  - For every atom T, except '\_', and for every term X, (T, X) belongs to Sets
-    (_atomic sets_).
-  - (\['\_'], []) belongs to Sets (the _untyped empty set_).
-  - For every tuple T = \{T\[1], ..., T\[n]\} and for every tuple X =
-    \{X\[1], ..., X\[n]\}, if (T\[i], X\[i]) belongs to Sets for every
-    1 <= i <= n, then (T, X) belongs to Sets (_ordered sets_).
-  - For every term T, if X is the empty list or a non-empty sorted list
-    \[X[1], ..., X\[n]] without duplicates such that (T, X\[i]) belongs to Sets
-    for every 1 <= i <= n, then (\[T], X) belongs to Sets (_typed unordered
-    sets_).
+[](){: #sets_definition } The sets recognized by this module are represented
```

Comment:
As you have introduced a header here, it is better to refer to that header than to have a special anchor. So all linkes to `#sets_definition` should be changed to `#sets-handled-by-this-module`.